src/super_gradients/examples/coco_eval_script/COCO_eval_script.py

```diff
@@ -0,0 +1,748 @@
+import json
```

Comment:
I think this script went into the PR by mistake...

Reply:
removed.